core/tests/puppeteer-acceptance-tests/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Accpetance Test for a Blog Admin to assign roles and change
+ * tag properties from the blog-admin page.
+ */
+
+const createNewUser = require(
+  '../puppeteer-testing-utilities/initializeUsersUtils.js');
+const { closeAllBrowsers } = require(
+  '../puppeteer-testing-utilities/initializeUsersUtils.js');
+
+const ROLE_BLOG_ADMIN = 'blog admin';
+const ROLE_BLOG_POST_EDITOR = 'blog post editor';
+
+let assignRolesToUsersAndChangeTagProperties = async function() {
+  const blogAdmin = await createNewUser.blogAdmin('blogAdm');
+  const superAdmin = await createNewUser.superAdmin('superAdm');
+  await createNewUser.guestUser('guestUsr1', 'guest_user1@example.com');
+  await createNewUser.guestUser('guestUsr2', 'guest_user2@example.com');
+
+  await blogAdmin.assignUserAsRoleFromBlogAdminPage('guestUsr1', 'BLOG_ADMIN');
+  await superAdmin.expectUserToHaveRole('blogAdm', ROLE_BLOG_ADMIN);
```

Comment:
Why is this needed? You've already created the user blogAdm with blog admin rights, and testing those isn't part of this test.

Shouldn't you be checking guestUsr1? Also, shouldn't the blogAdmin be doing that, not the superadmin? The superadmin should not appear in this test at all.

Reply:
ohh thanks for pointing out.. this expectation must be to check the blog admin rights to the guestUsr1
and this thing must be done by the superAdmin because only the super admin is able to visit the /admin page, any other user (not signed in with the email `testadmin@example.com`) cannot visit the /admin page, so thats why we need superAdmin here to check the roles..

Reply:
Can the blog admin see the list of blog editors in the blog admin page (rather than the /admin page)? If so, that would be better.

(If not, what does the blog admin page look like? It seems like an omission to me if they can add reviewers but not see reviewers -- /cc @Rijuta-s @gp201)

Reply:
we cannot see the list of blog editors in the /blog-admin page.